runtime/sema/entrypoint.go

```diff
@@ -24,30 +24,6 @@ import (
 
 const FunctionEntryPointName = "main"
 
-// FunctionEntryPointType returns the type of the entry point function declaration, if any.
-//
-// Returns an error if no valid entry point function declaration exists.
-//
-func (checker *Checker) FunctionEntryPointType() (*FunctionType, error) {
```

Comment:
Moved to elaboration, as this function only needs the elaboration, not the full checker